vending_machine_control/DrinkType[] drink_types
int32[] drink_amounts

---

vending_machine_control/DrinkType[] delivered_drink_types
int32[] delivered_drink_amounts

vending_machine_control/ErrorCodes[] error_code

---

std_msgs/String order_status